features/support/env.rb

```diff
@@ -18,3 +18,6 @@
   Capybara::Poltergeist::Driver.new(app, js_errors: false)
 end
 Capybara.javascript_driver = :poltergeist
+
+Warden.test_mode!
+World Warden::Test::Helpers
```

Comment:
Okay so you added the Warden helpers. That's great. But I can't see that you actually use them.  What's up?